### ISSUE TITLE
Zooming and Panning works weird on mobile

### ISSUE THREAD

**Issue by SnowSea-Dream** [mapbox/mapbox-gl-draw#887]
**mapbox-gl-js version**:
**mapbox-gl-draw version**:

### Steps to Trigger Behavior

 1.
 2.
 3.

**Comment by SnowSea-Dream** [comment]
Zooming and Panning works weird on mobile

**Comment by SnowSea-Dream** [comment]
Zooming and Panning starts a little slowly on mobile before integrating this plugin.

**Comment by SnowSea-Dream** [comment]
If you zoom the map on the phone it first moves the map before starting to zoom it.

**Comment by aptroost** [comment]
Agree, for me it occurs on the official [example](https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/) with iPhone X iOS 12.4, both on Chrome and Safari browsers.
